content/references/protocol-reference/transactions/transaction-types/accountset.md

```diff
@@ -58,8 +58,8 @@ You can put any domain in your account's `Domain` field. To prove that an accoun
 
 There are several options which can be either enabled or disabled for an account. Account options are represented by different types of flags depending on the situation:
 
-* The `AccountSet` transaction type has several "AccountSet Flags" (prefixed **`asf`**) that can enable an option when passed as the `SetFlag` parameter, or disable an option when passed as the `ClearFlag` parameter.
-* The `AccountSet` transaction type has several transaction flags (prefixed **`tf`**) that can be used to enable or disable specific account options when passed in the `Flags` parameter. This style is discouraged. New account options do not have corresponding transaction (`tf`) flags.
+* The `AccountSet` transaction type has several "AccountSet Flags" (prefixed **`asf`**) that can enable an option when passed as the `SetFlag` parameter, or disable an option when passed as the `ClearFlag` parameter. Some options have only this style of flag. You can only enable up to one `asf` flag per transaction.
```

Comment:
"Some options have only this style of flag" sounds ambiguous. Could we clarify that we're referring to newer options?